resources/deb/install-ubuntu.sh

```diff
@@ -13,17 +13,13 @@
     # run inside sudo
     $SUDO sh <<SCRIPT
   set -ex
-  apt-get update
-  export APT_TRANSPORT_MISSING=$(dpkg -l apt-transport-https | grep -c 'no packages')
-  if [ $APT_TRANSPORT_MISSING == '1' ]; then
-    apt-get install apt-transport-https
-  fi
+  dpkg -s apt-transport-https 1>/dev/null 2>/dev/null || (apt-get update && apt-get install -y apt-transport-https)
```

Comment:
The thing with apt-get is it will not install the package if it is already installed and the latest version. So we really don't even need the check.